Sources/Authgear.swift

```diff
@@ -672,22 +715,49 @@ public class Authgear {
     public func application(
         _ application: UIApplication,
         continue userActivity: NSUserActivity,
-        restorationHandler: @escaping ([UIUserActivityRestoring]?) -> Void
+        restorationHandler: @escaping ([UIUserActivityRestoring]?) -> Void,
+        handler: VoidCompletionHandler? = nil
     ) -> Bool {
+        var matched = false
         if userActivity.activityType == NSUserActivityTypeBrowsingWeb,
            let url = userActivity.webpageURL {
-            _ = handleWechatRedirectURI(url)
+            matched =
+                handleWechatRedirectURI(url, handler: handler) ||
+                handleLoginLinkVerificationURI(url, handler: handler)
         }
-        return true
+        return matched
     }
 
     @available(iOS 13.0, *)
-    public func scene(_ scene: UIScene, continue userActivity: NSUserActivity) {
+    public func scene(
+        _ scene: UIScene,
+        continue userActivity: NSUserActivity,
+        handler: VoidCompletionHandler? = nil
+    ) -> Bool {
         guard userActivity.activityType == NSUserActivityTypeBrowsingWeb,
               let url = userActivity.webpageURL else {
-            return
+            return false
         }
-        _ = handleWechatRedirectURI(url)
+        var matched = false
+        matched =
+            handleWechatRedirectURI(url, handler: handler) ||
+            handleLoginLinkVerificationURI(url, handler: handler)
+        return matched
+    }
+
+    @available(iOS 13.0, *)
+    public func scene(
+        _ scene: UIScene,
+        openURLContexts URLContexts: Set<UIOpenURLContext>,
+        handler: VoidCompletionHandler? = nil
```

Comment:
ref: https://developer.apple.com/documentation/xcode/supporting-universal-links-in-your-app

From the doc, to handle universal link we should use `scene(_:willConnectTo:options:)`  and `scene(_:continue:)`? Can you help check and confirm it? 🙏🏻